### PR TITLE
Add table alicloud_account. Closes #174

### LINKED ISSUE #174: Add table alicloud_account
**References**
Add any related links that will help us understand the resource, including vendor documentation, related GitHub issues, and Go SDK documentation.

### PR DESCRIPTION
**Pull request by khushboo9024**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
No env file present for the current environment:  staging 
 Falling back to .env config
No env file present for the current environment:  staging
customEnv TURBOT_TEST_EXPECTED_TIMEOUT undefined

SETUP: tests/alicloud_account []

PRETEST: tests/alicloud_account

TEST: tests/alicloud_account
Running terraform
data.alicloud_account.current: Refreshing state...

Apply complete! Resources: 0 added, 0 changed, 0 destroyed.

Outputs:

current_account_id = ******

Running SQL query: test-list-query.sql
[
  {
    "account_id": "*****",
    "akas": [
      "arn:acs:::****"
    ],
    "alias": "****"
  }
]
✔ PASSED

Running SQL query: test-not-found-query.sql
null
✔ PASSED

Running SQL query: test-turbot-query.sql
[
  {
    "akas": [
      "arn:acs:::****"
    ],
    "alias": "****",
    "title": "****"
  }
]
✔ PASSED

POSTTEST: tests/alicloud_account

TEARDOWN: tests/alicloud_account

SUMMARY:

1/1 passed.


```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
> select
  alias,
  account_id,
  akas,
  title
from
  alicloud_account;

+------------------+------------------+--------------------------------+------------------+
| alias                        | account_id             | akas                                              | title                        |
+------------------+------------------+--------------------------------+------------------+
| ****************.    | ****************     | ["arn:acs:::****************"]      | ****************.    |
+------------------+------------------+--------------------------------+------------------+

```
</details>

### CODE REVIEW

**Review by Subhajit97 — APPROVED** [review]
LGTM

**Review by rajmohanty17 — APPROVED** [review]
LGTM

File: docs/tables/alicloud_account.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Table: alicloud_account
+
+The Alibaba Cloud account is a container for your Alibaba Cloud resources. You create and manage your Alibaba Cloud resources in an Alibaba Cloud account.It has two accounts: International-Site Account and China-Site Account. In most cases the account type makes no difference when creating Alibaba Cloud resources.
+
+## Examples
+
+### Basic AliCloud account info
```
Comment (cbruno10) [inline docs/tables/alicloud_account.md:7]:
```suggestion
### Basic info
```

File: docs/tables/alicloud_account.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Table: alicloud_account
+
+The Alibaba Cloud account is a container for your Alibaba Cloud resources. You create and manage your Alibaba Cloud resources in an Alibaba Cloud account.It has two accounts: International-Site Account and China-Site Account. In most cases the account type makes no difference when creating Alibaba Cloud resources.
```
Comment (cbruno10) [inline docs/tables/alicloud_account.md:3]:
```suggestion
The Alibaba Cloud account is a container for your Alibaba Cloud resources. You create and manage your Alibaba Cloud resources in an Alibaba Cloud account.
```

File: alicloud/table_alicloud_account.go [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+func tableAlicloudAccount(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "alicloud_account",
+		Description: "Alicloud Account",
+		List: &plugin.ListConfig{
+			Hydrate: listAccountAlias,
+		},
+		Columns: []*plugin.Column{
+			{
+				Name:        "alias",
```
Comment (cbruno10) [inline alicloud/table_alicloud_account.go:24]: Is this the only property we get back about the account?

**Review by cbruno10 — CHANGES_REQUESTED** [review]
Please see comments

File: alicloud/table_alicloud_account.go (hunk 3 above)
Comment (Subhajit97) [reply alicloud/table_alicloud_account.go:24]: @cbruno10 Currently the API returns only `AccountAlias` in response
https://www.alibabacloud.com/help/doc-detail/28737.htm

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**